steam.py

```diff
@@ -146,21 +148,51 @@ def check_game_sales():
     # Fetch all price_overviews
     url = f'https://store.steampowered.com/api/appdetails?filters=price_overview&appids={app_ids_string}'
     result = requests.get(url)
-    games_on_sale = {}
+    sales = {}
+    new_sales = {}
     if result.status_code == 200: 
         sale_data = json.loads(result.text)
         # TODO: clean up loop with map
         #loop through all data and extract price overview
+        
         for app_id in app_ids: 
             price_overview = sale_data[app_id]['data']['price_overview']
-            # If game is on sale, update wishlist and track sale games
-            if price_overview['discount_percent'] > 0:
+            sale_history = wishlist_json[app_id]['sale_history']
+            on_sale_yesterday = sale_history['sale_start'] != None
+            on_sale = price_overview['discount_percent'] > 0

```

Comment:
Does this return True False?

Reply:
Yeah, just turning our check into a Boolean for readability, we are using the raw check in the if statement in the other task

Reply:
Boolean = Variable that is true or false

Reply:
schweet got it thanks